prow/plugins/trigger/trigger.go

```diff
@@ -49,6 +49,7 @@ type githubClient interface {
 	GetPullRequestChanges(org, repo string, number int) ([]github.PullRequestChange, error)
 	RemoveLabel(org, repo string, number int, label string) error
 	DeleteStaleComments(org, repo string, number int, comments []github.IssueComment, isStale func(github.IssueComment) bool) error
+	GetIssueLabels(org, repo string, number int) ([]github.Label, error)
```

Comment:
This is unused.

Reply:
Yeah this is unused now. I was thinking of using this func in pr.go.
Thanks!